$common/fragments/config.py

```diff
@@ -15,6 +15,7 @@ def config_file_to_dict(file_path):
         "plagiarism": False,
         "external_libraries": None,
         "custom_feedback_script": None,
+        "username_for_full_log": None,
```

Comment:
Maybe this should be a list of usernames? I do not see any reason to not allow multiple persons to have the full logs (for instance if there is multiple TA, which is the case at the moment:-))

Reply:
Why not ? I will update the code